change the wording for a smoother word & making it clear it's not a common thing (#855)
meilisearch/documentation: learn/advanced/asynchronous_updates.md

}
```

## Terminate MeiliSearch While a Task is Processing

> In the rare cases where you would need to terminate MeiliSearch, since the asynchronous tasks are <clientGlossary word="atomic"/>, terminating MeiliSearch in the middle of a process does not corrupt or alter the database.

Reviewer: ```suggestion
In the rare cases when you need to terminate MeiliSearch in the middle of a process, know that it will not corrupt or alter the database in any way. This is because MeiliSearch's asynchronous tasks are <clientGlossary word="atomic"/>.
```
